cli/build/main.rs

```diff
@@ -195,10 +200,14 @@ fn do_main() -> Result<(), Error> {
 
 	let mut ctor_module = module.clone();
 
+	let mut public_api_entries = matches.value_of("public_api")
+		.map(|val| val.split(",").collect())
```

Comment:
haven't you tried [this](https://docs.rs/clap/2.29.0/clap/struct.Arg.html#method.multiple)?

Reply:
like it is said there, it does not play well with positional arguments which we have